admin/README.md

```diff
@@ -0,0 +1,3 @@
+# Admin fragments
```

Comment:
Ignore the stuff in ``admin/``; I started to implement some things that were out of scope, and don't want to lose them.